_locales/it/messages.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": {
+    "message": "Tabs aside",
```

Comment:
"Tabs aside" in Italian is translated as "Raccolta", which means "collection" in Italian, so I preferred not to translate it and leave "Tabs aside"

Reply:
Well, initially, "Tabs aside" feature has also been referred as "Tab groups", so perhaps this will be more suitable for translation.

By the way, this extension derives from EdgeHTML Microsoft Edge and you can try to check how it has been named there.